libraries/AC_InputManager/AC_InputManager_Heli.cpp

```diff
@@ -119,4 +122,34 @@ float AC_InputManager_Heli::get_pilot_desired_collective(int16_t control_in)
     return collective_out;
 }
 
+// parameter_check - check if input manager specific parameters are sensible
+bool AC_InputManager_Heli::parameter_check(char* fail_msg, uint8_t fail_msg_len) const
```

Comment:
I'm guessing that additional constraints could be put in here - low and high between min and max, a strict ascending or descending ordering, that sort of thing?